engines/bastion/app/assets/stylesheets/bastion/overrides.scss

```diff
@@ -77,6 +77,7 @@ html .toolbar-pf.table-view-pf-toolbar-external .toolbar-pf-filter {
 .modal-body {
   overflow-x: hidden;
   overflow-y: scroll;
+  max-height: 75vh; // prevent modal from going below current view causing two scrollbars
```

Comment:
The modal will go below the current viewport height causing two scrollbars and part of it to be hidden. This will limit the body of it to 75% of the viewport and the modal will be fixed. The user can then scroll just the modal. I changed it for the long list of packages in the matching filter modal, but here is another example for the host collections page:

Before (notice there are two scrollbars and much of the modal is hidden):
![here](http://www.clipular.com/c/4747545568608256.png?k=lXFo2JN_nuNtHNhCJfJUR71_DAQ)

After, modal is in full view and the only scroll is in the modal:
![here](http://www.clipular.com/c/4971914324606976.png?k=QyAjLPVi3PMMplqvldMLZKTo16M)